ucc/project/loadbalancer/LoadBalancerTest.java

```diff
@@ -1,4 +1,4 @@
-package ucc.project.loadbalancer;
+package org.example.loadbalancer;
```

Comment:
this is not correct. No need to touch loadbalancer class for now. package ucc.project.loadbalancer is correct